### PR TITLE
MML adjust to new firing circle

### PR DESCRIPTION
**Pull request by Apofenas**

### CODE REVIEW

File: units/INU2003/INU2003_unit.bp [hunk 1]
```diff
@@ -302,7 +302,7 @@ UnitBlueprint {
             TurretBoneMuzzle = 'muzzle.001',
             TurretBonePitch = 'barrel',
             TurretBoneYaw = 'turret',
-            TurretPitch = 120,
+            TurretPitch = 125, --almost vertical! if only we had proper Retard models to replace retarded model...
```
Comment (CookieNoob) [inline units/INU2003/INU2003_unit.bp]: what does this comment add to the bp?

File: projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_Script.lua [hunk 2]
```diff
@@ -11,12 +11,12 @@ NTacticalMissile1_HighArc = Class(ArcingTacticalMissile) {
     end,
 
     StageThread = function(self)
-        WaitSeconds(1.1)
+        WaitSeconds(4)  --2d stage missile valley splits via zigzags and turns towards target
```
Comment (CookieNoob) [inline projectiles/NTacticalMissile1_HighArc/NTacticalMissile1_HighArc_Script.lua]: a missile valley?

either remove these comments or fix the spelling errors

File: units/INU2003/INU2003_unit.bp [hunk 3]
```diff
@@ -141,9 +141,9 @@ UnitBlueprint {
         UniformScale = 0.12,
     },
     Economy = {
-        BuildCostEnergy = 1650,
-        BuildCostMass = 220,
-        BuildTime = 1100,
+        BuildCostEnergy = 2500,
+        BuildCostMass = 300,
```
Comment (CookieNoob) [inline units/INU2003/INU2003_unit.bp]: this is a pretty high increase in cost for this unit

**Comment by CookieNoob** [comment]
you dont need to make a new commit for every file you change, but ideally for every change in total. I.e. you change the mml and have to adjust 3 files, then this goes into one commit. You then change some other unit -> new commit (or if it is related to the first change -> same commit)

changes to the slugger are not mml related probably, why are they here?

File: units/INU2003/INU2003_unit.bp (hunk 3 above)
Comment (Apofenas) [reply units/INU2003/INU2003_unit.bp]: This is reasonable increace to a unit that shoots 4 missiles at once and can cross water

Comment (CookieNoob) [reply units/INU2003/INU2003_unit.bp]: yes, but this should have been part of the original change already imo

**Comment by CookieNoob** [comment]
no need to close the PR, just remove the commits that dont have anything to do with the thing you want to change...
I will squash the commits into one commit before I merge it. The comment was about future changes.

**Pull request closed without merging**